nmdc_runtime/api/endpoints/queries.py

```diff
@@ -127,6 +133,28 @@ def _run_query(query, mdb) -> CommandResponse:
                     status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
                     detail="Failed to back up to-be-deleted documents. operation aborted.",
                 )
+    elif q_type is UpdateCommand:
+        collection_name = query.cmd.update
+        if collection_name not in nmdc_schema_collection_names(mdb):
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail="Can only update documents in nmdc-schema collections.",
+            )
+        update_specs = [
```

Comment:
I don't understand why this is named `update_specs` while the (pre-existing) one for the `DeleteCommand` is named `find_specs`. I think they are equally "find" spec (i.e. finding the thing we want to either update or delete).

Reply:
I updated `find_specs` to `delete_specs` for consistency and updated a few var names for clarity